lib/roots/recipe.ex

```diff
@@ -1,7 +1,9 @@
 defmodule Roots.Recipe do
   use Roots.Model
+  
   alias Roots.{Repo, Recipe}
 
+
   schema "recipes" do
     field :description, :string
```

Comment:
Do we want to make this text, instead of a string. I imagine we might have some lengthy recipe descriptions.